CHANGELOG.md

```diff
@@ -8,220 +8,223 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* Add support for editing the podspec, license, README, license, and docs of local development pods  
+* Add color indication to output of `pod outdated`
```

Comment:
uh this does not look like intended right?

Reply:
Oh you're right, VSCode did some unwanted auto formatting on change there